homeassistant/components/whois/sensor.py

```diff
@@ -81,7 +86,7 @@ def _ensure_timezone(timestamp: datetime | None) -> datetime | None:
         name="Admin",
         icon="mdi:account-star",
         entity_category=EntityCategory.DIAGNOSTIC,
-        entity_registry_enabled_default=True,
+        entity_registry_enabled_default=False,
```

Comment:
Should this change be marked as a breaking change?

Reply:
This have only existed in dev, was added yesterday